src/lexer/tokens.rs

```diff
@@ -159,6 +159,9 @@ pub enum Token {
     #[token("=>")]
     KeywordOutputAssignment,
 
+    #[token("REF=")]
+    KeywordReferenceAssignment,
+
```

Comment:
How strict do we want to be here, i.e. should spaces between REF and = be allowed or no?

Reply:
I'd treat this the same as e.g. `: =`